microsoft-edge/webview2/get-started/wpf.md

```diff
@@ -77,6 +77,7 @@ Start with a basic desktop project that contains a single main window.
 1. Enter values for **Project name** and **Location**.
 
 1. In the **Framework** dropdown list, select **.NET Framework 4.6.2** or later.
+   <!-- TODO: are Core and Framework both supported?  eg clarify a statement like: Core and Framework are both supported. -->
```

Comment:
@nishitha-burman - are Core and Framework both supported?
Rendered section:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-2135#step-3---create-a-single-window-webview2-app